pyproject.toml

```
@@ -5,12 +5,145 @@ requires = [
     "setuptools>=51.0.0",
```

```suggestion
    "setuptools>=61.0.0",
```

You *must* use a version of setuptools new enough to support using pyproject.toml metadata. This is the first version in which experimental support was added -- I have not checked if that is even sufficient, older versions may be buggy enough that you need to require an even higher minimum version, but this is the lowest version that it's even worth attempting to look at.

---

Thanks for the catch.